source/upgrade-guide/upgrading-central-components.rst

```diff
@@ -17,28 +17,36 @@ This section guides you through the upgrade process of the Wazuh indexer, the Wa
 
    Apache Lucene does not support downgrades, meaning once you upgrade to Wazuh 4.12.0 or later, you cannot roll back to 4.11 and earlier versions without a fresh installation of the indexer.
 
-   To avoid data loss, create an :ref:`index snapshot <migrating_indices_take_snapshots>` before upgrading. For more details, refer to the `Opensearch documentation <https://opensearch.org/docs/latest/install-and-configure/upgrade-opensearch/rolling-upgrade/#:~:text=Important%3A%20OpenSearch%20nodes%20cannot%20be,before%20beginning%20the%20upgrade%20procedure.>`__.
+   To avoid data loss, create an :ref:`index snapshot <migrating_indices_take_snapshots>` before upgrading. For more details, refer to the `Opensearch documentation <https://opensearch.org/docs/latest/install-and-configure/upgrade-opensearch/rolling-upgrade/>`__ for more information.
```

Comment:
This sounds weird `For more details, refer to... for more information` I would rather remove the `for more information` ending.